### PR TITLE
Fix input file path

### LINKED ISSUE #2253: Missing scream init file on ANL server
`map_ne4np4_to_ne2np4_mono.nc` is missing from the public e3sm site:

```
  Could not download scream/init/map_ne4np4_to_ne2np4_mono.nc, output was:
  WARNING:root:wget failed with output: and errput --2023-03-20 22:50:46--
  https://web.lcrc.anl.gov/public/e3sm/inputdata/atm/scream/init/map_ne4np4_to_ne2np4_mono.nc


  Resolving web.lcrc.anl.gov (web.lcrc.anl.gov)...  140.221.70.30

  Connecting to web.lcrc.anl.gov (web.lcrc.anl.gov)|140.221.70.30|:443...
  connected.

  HTTP request sent, awaiting response...  404 Not Found

  2023-03-20 22:50:47 ERROR 404: Not Found.
```

### PR DESCRIPTION
**Pull request by elynnwu**
This PR updates the location of`map_ne4np4_to_ne2np4_mono.nc`. It currently looks under the `init` directory and causes a file not found error while trying to download form ANL server if `SCREAM_INPUT_ROOT` is not specified.

Fixes #2253.

### CODE REVIEW

File: components/eamxx/src/physics/rrtmgp/CMakeLists.txt [hunk 1]
```diff
@@ -164,6 +164,8 @@ set (RRTMGP_TABLES
   scream/init/rrtmgp-allsky.nc
   scream/init/rrtmgp-cloud-optics-coeffs-sw.nc
   scream/init/rrtmgp-cloud-optics-coeffs-lw.nc
+  scream/init/rrtmgp-data-sw-g224-2018-12-04.nc
```
Comment (AaronDonahue) [inline components/eamxx/src/physics/rrtmgp/CMakeLists.txt:167]: Why are we adding these files too?  They seem unrelated to the PR.

**Review by AaronDonahue — COMMENTED** [review]
Looks good, but I had a question about the rrtmgp files.

File: components/eamxx/src/physics/rrtmgp/CMakeLists.txt (hunk 1 above)
Comment (elynnwu) [reply components/eamxx/src/physics/rrtmgp/CMakeLists.txt:167]: Right, this is a file requested by generate baseline. For machines with all the correct files, no error will be thrown since it just checks that the file is there then read it.

Comment (AaronDonahue) [reply components/eamxx/src/physics/rrtmgp/CMakeLists.txt:167]: okay sounds good, thanks for checking.

**Review by AaronDonahue — APPROVED** [review]
